whylog/config/__init__.py

```diff
@@ -119,14 +138,26 @@ def _load_parsers(self):
         )
 
     def _load_rules(self):
-        return [
-            RegexRuleFactory.from_dao(serialized_rule, self._parsers)
-            for serialized_rule in self._load_file_with_config(self._rules_path)
-        ]
+        loaded_rules = defaultdict(list)
+        for serialized_rule in self._load_file_with_config(self._rules_path):
+            loaded_rule = RegexRuleFactory.from_dao(serialized_rule, self._parsers)
```

Comment:
Name `loaded_rule` is very similar to `loaded_rules`. It may confuse reviewer and future developers, and simple typo may cause huge problems.